arbstate/inbox.go

```diff
@@ -72,12 +73,23 @@ func parseSequencerMessage(ctx context.Context, data []byte, das DataAvailabilit
 			if das == nil {
 				log.Error("No DAS configured, but sequencer message found with DAS header")
 			} else {
-				var err error
-				payload, err = das.Retrieve(ctx, data[40:])
+				cert, err := DeserializeDASCertFrom(bytes.NewReader(data[40:]))
 				if err != nil {
-					log.Error("Reading from DAS failed", "err", err)
+					log.Error("Deserializing data availability cert failed", "err", err)
+					return nil
+				}
+				if err := cert.VerifyNonPayloadParts(ctx, das); err != nil { // safe because L1 verified keyset hash
+					log.Error("Invalid data availability cert", "err", err)
+					return nil
```

Comment:
This can also fail because of an invalid signature, in which case we don't want to fail to read the message, instead replacing it with an empty one

Reply:
I think we want to return an error here if the signature on the DA Cert is bad.

Reply:
We should log an error, but we don't want to pause the chain if a batch is submitted with an invalid DAS signature

Reply:
Changed how we handle these invalid conditions, so we don't return an error but instead return as if there was a valid batch containing no segments.